float64 timestamp
uint32 type
uint32 nodeID
uint32 nodeType
amee/Pose pose